### DISCUSSION TITLE
Room Card: Extend color_on support to icon background

### DISCUSSION THREAD

**Discussion by lyneld** [andyblac/UI-Minimalist-Custom-Cards#25] [Ideas]
Would be great to have the color_on feature of the room card extended to the icon background as well. 

Currently, the “ulm_custom_card_andyblac_room_color_on” variable only affects the room icon. Having an option to change the background color as well would be nice.

**Comment by andyblac** [comment]
what would you expect to happen with sensor icons in active state ?, as they have there own state control and colour on.
off state
<img width="294" alt="Screenshot 2024-08-22 at 13 55 50" src="https://github.com/user-attachments/assets/d5853380-9bb4-47af-862b-9d7dbb9dac0a">
on state
<img width="277" alt="Screenshot 2024-08-22 at 13 55 58" src="https://github.com/user-attachments/assets/29eed3d2-3bdb-4528-8405-68f75d9c4664">

**Comment by andyblac** [comment]
or we can background card colour instead ?

<img width="287" alt="Screenshot 2024-08-22 at 18 54 31" src="https://github.com/user-attachments/assets/4a636a45-dc8c-4f1e-b9e0-45ad7a5e9435">

**Comment by lyneld** [comment]
Ah interesting! I hadn’t really thought about the sensor icons since I don’t use them as of now. But might do eventually. I like the idea of the card lighting up when any of the entities are on. 

Its probably overkill, but maybe an option to have any combination of:
1. Icon light up (already exists) 
2. Icon background light up
3. Card light up

Sensor icons (I assume you mean the six icons around the big icon) would light up individually without impacting the room card color.
